test/test_eds.py

```diff
@@ -1,6 +1,7 @@
 import os
 import unittest
 import canopen
+from canopen.objectdictionary import ObjectDictionaryError
```

Comment:
No longer needed.

Reply:
Good catch; thanks!

Reply:
Addressed in 9490fbf06dcf92632790d6d4b9f966eda9105d4e; marking as resolved.